docs/monitors/disk.md

```diff
@@ -0,0 +1,86 @@
+<!--- GENERATED BY gomplate from scripts/docs/monitor-page.md.tmpl --->
+
+# disk
+
+
+This monitor reports metrics about free disk space.
+
+```yaml
+monitors:
+ - type: disk
+```
+
+
+Monitor Type: `disk`
+
+[Monitor Source Code](https://github.com/signalfx/signalfx-agent/tree/master/internal/monitors/disk)
+
+**Accepts Endpoints**: No
+
+**Multiple Instances Allowed**: **No**
+
+## Configuration
+
+| Config option | Required | Type | Description |
+| --- | --- | --- | --- |
+| `procFSPath` | no | `string` | The path to the proc filesystem. Useful to override in containerized environments.  (Does not apply to windows) (**default:** `/proc`) |
+| `disks` | no | `list of string` | Which devices to include/exclude (**default:** `[/^loop[0-9]+$/ /^dm-[0-9]+$/]`) |
+| `ignoreSelected` | no | `bool` | If true, the disks selected by `disks` will be excluded and all others included. |
```

Comment:
Same thing.  This flag that makes the `disks` option mean the exact opposite is so confusing.  Something like what we do with metric filters would be better, have a `disks` and `excludedDisks` or something.

Reply:
ack